tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -0,0 +1,946 @@
+#!/usr/bin/python
+""" The Python Hipify script.
+##
+# Copyright (c) 2015-2016 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
@soumith re. copyright

Reply:
once we make modifications (going forward), we can add a copyright line that says "portions copyright AMD, portions copyright Facebook"

Reply:
updated copyright, now also reflects correctly that both AMD and Facebook have contributed already to pyHIPIFY